lib/workers/repository/update/branch/index.ts

```diff
@@ -540,14 +544,6 @@ export async function processBranch(
 
       // compile commit message with body, which maybe needs changelogs
       if (config.commitBody) {
-        if (
-          config.fetchReleaseNotes &&
-          needsChangelogs(config, ['commitBody'])
```

Comment:
@rarkins should we throw an error If changelogs are required but set to PR or off?

Reply:
No, I'd like to drop this "needsChangelogs()" if we onlyuse it for this